client/src/components/Home.js

```diff
@@ -1,13 +1,43 @@
 import React from "react";
 import "../styles/Home.css";
 import { Header } from "semantic-ui-react";
```

Comment:
You can put these two lines in one

Reply:
Hi Islam, thanks for your comments, could you please tell me which two lines?are they  2, and 3 ?

Reply:
2 lines that are similar to each other :)